src/components/FormMessage.tsx

```diff
@@ -0,0 +1,167 @@
+import Button from '@sendbird/uikit-react/ui/Button';
+import Label, {
+  LabelTypography,
+  LabelColors,
+} from '@sendbird/uikit-react/ui/Label';
+import { useCallback, useState, useEffect } from 'react';
+// eslint-disable-next-line import/no-unresolved
+import { EveryMessage } from 'SendbirdUIKitGlobal';
+import styled from 'styled-components';
+
+import Input from './FormInput';
+
+const Root = styled.div`
+  max-width: 244px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px 12px;
+  gap: 8px;
+  border-radius: 16px;
+  background-color: #eeeeee;
+`;
+
+const SubmitButton = styled(Button)`
+  width: 100%;
+`;
+
+interface Field {
+  key: string;
+  title: string;
+  placeholder: string;
+  required: boolean;
+  regex: RegExp;
+  input_type: string;
+}
+interface Props {
+  message: EveryMessage;
+  form: {
+    key: string;
+    fields: Field[];
+    /** ubmitted data */
+    data: Record<string, string>;
+  };
+}
+
+type FormValues = Record<
+  string,
+  { value: string; required: boolean; hasError: boolean; isValid: boolean }
+>;
+
+export default function FormMessage(props: Props) {
+  const {
+    message,
+    form: { fields, key: formKey, data: submittedData },
```

Comment:
I would define an interface for
`{ fields, key: formKey, data: submittedData }`

Reply:
It was in the above. The Form interface. 
https://github.com/sendbird/chat-ai-widget/pull/60/commits/e9ee8babf3a0ac93fc59d72635f4d71572038a9a#diff-36253251c537f70c554e58aa06a78b18c0d26a5f973c8c1641c292434c7ffe31R22

Reply:
Nice can we replace with it?

Reply:
Yeah it's been applied. The attached link is a commit link.